test/images/README.md

```diff
@@ -0,0 +1,25 @@
+This image is the base image for Kubeflow Pipelines [Prow job](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubeflow/kubeflow-postsubmits.yaml#L245). Including presubmit and postsubmit jobs.
```

Comment:
It's always preferrable to pin the link to a commit (aka permalink), to prevent potential broken links.